tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.scss

```diff
@@ -78,7 +83,8 @@ $_title-to-heading-gap: 12px;
 }
 
 .chart-container {
-  flex-grow: 1;
+  overflow: hidden;
+  resize: vertical;
```

Comment:
I am just going to comment and say I am slightly leaning towards removing this. However, my preference is not strong so I leave it to you.